plugins/chat/app/services/chat/channel/policy/message_creation.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class Chat::Channel::MessageCreationPolicy < PolicyBase
+class Chat::Channel::Policy::MessageCreation < Service::PolicyBase
```

Comment:
We should probably apply that `Chat::Channel` namespace to related services, I’ll do that in another PR :sweat_smile: